doc/source/template_nocontainer_nextflow.config

```diff
@@ -12,12 +12,18 @@ process {
     // Specify memory, number of threads and modules to be loaded for processes submitted to the cluster 
     // It is advised to provide higher computational power to wgdParalogs and wgdOrthologs 
     withName: 'wgdParalogs' {
-        clusterOptions = ''  // e.g. '-pe serial 10 -l h_vmem=2G'
+        cpus = ''            // e.g. '8'
+        penv = ''            // e.g. 'serial'
+        memory = ''          // e.g. '16GB'
+        clusterOptions = ''  // e.g. '-l h_vmem=2G'
         beforeScript = ''    // e.g. 'module load python paml muscle mafft prank fasttree blast mcl i-adhore'
     }
     
     withName: 'wgdOrthologs' {
-        clusterOptions = ''  // e.g. '-pe serial 10 -l h_vmem=2G'
+        cpus = ''            // e.g. '8'
+        penv = ''            // e.g. 'serial'
+        memory = ''          // e.g. '16GB'
+        clusterOptions = ''  // e.g. '-l h_vmem=2G'
         beforeScript =  ''   // e.g. 'module load python paml muscle mafft prank blast mcl'
```

Comment:
See above for mafft and prank, also wgd ortholog analyses don't need mcl.

Reply:
Agreed!